src/search/components/JsonData/style.scss

```diff
@@ -0,0 +1,7 @@
+.JsonData .JsonData-json {
+  display: none;
+}
+
+.JsonData.JsonData-visible .JsonData-json {
```

Comment:
Looking into the likes of BEM, and other CSS methodologies etc, the idea is that you'd use something more like this:

```css
.JsonData-json {} // specifity 10
.JsonData-visible .JsonData-json {} // specifity 20
```

instead of:

```css
.JsonData .JsonData-json {} // specifity 20
.JsonData.JsonData-visible .JsonData-json {} // specifity 30
```

You could probably re-jig things even more to get it down to a single className which controls the state e.g something like this maybe?:

```css
.JsonData-json {}
.JsonData-json-visible {}
```

This simplifies things even further but it requires a bit of a change in terms of how we think about CSS. 

I'm not advocating we use a specific methodology just yet, but I think it's worth exploring it.